packages/test-apps/tsconfig.json

```diff
@@ -0,0 +1,13 @@
+{
+  "extends": "../../tsconfig.json",
+  "include": ["."],
```

Comment:
This might accidentally include `node_modules` for the test apps, probs good to add that to the excludes

Reply:
Nice catch, thanks!